src/components/Ranking.tsx

```diff
@@ -0,0 +1,168 @@
+import type { ThemeType } from "@/styles/theme";
+import { useTheme } from "@emotion/react";
+import styled from "@emotion/styled";
+import { useState } from "react";
+import RankingList from "@/components/RankingList";
+
+type TypeCategory = {
+  image: string;
+  name: string;
+  targetType: string;
+};
+
+const categoryList: TypeCategory[] = [
+  {
+    image: "ALL",
+    name: "전체",
+    targetType: "ALL",
+  },
+  {
+    image: "👩🏻",
+    name: "여성",
+    targetType: "FEMALE",
+  },
+  {
+    image: "👨🏻",
+    name: "남성",
+    targetType: "MALE",
+  },
+  {
+    image: "👦🏻",
+    name: "청소년",
+    targetType: "TEEN",
+  },
+];
+
+const Ranking = () => {
+  const [category, setCategory] = useState(0);
+  const [wishCategory, setWishCategory] = useState(0);
+  const theme = useTheme();
+  return (
+    <Container>
+      <Title>실시간 급상승 선물랭킹</Title>
+      <NavBar>
+        <CategoryList>
+          {categoryList.map((e, index) => (
+            <Category
+              key={index}
+              onClick={() => {
+                setCategory(index);
+              }}
+            >
+              <CategoryImg
+                selected={category == index ? true : false}
```

Comment:
자바스크립트에서 비교 연산자는 `===` 주로 사용하는데요,
두 연산자는 어떤 차이점을 가지고 있는지 찾아보는 것도 좋아보여요

Reply:
비교 연산자는 비교한 결과를 boolean 값으로 반환하기 때문에 아래와 같이 작성해 볼 수 있겠네요!

```suggestion
                selected={category === index}
```

Reply:
아래 반복해서 사용하는걸로 보이는데 함수로 분리할 수 있지 않을까요?